src/test/java/com/trello/registration/RegistrationTest.java

```diff
@@ -0,0 +1,104 @@
+package com.trello.registration;
+
+import static com.trello.Waiters.waitSeconds;
+
+import com.trello.Waiters;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.Assertion;
+
+import java.util.List;
+
+public class RegistrationTest {
+
+
+    public Actions action;
+
+//    @BeforeClass
+//    public void webDriver() {
```

Comment:
Please remove unused methods in all test classes

Reply:
ok